app/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx

```diff
@@ -57,7 +57,7 @@ export const AccountDetailsCard: React.FC<AccountDetailsCardProps> = ({
 
   return (
     <>
-      <HeadContentContainer isTruncated={isTruncated}>
+      <HeadContentContainer isTruncated={isTruncated} isMobile={isMobile}>
```

Comment:
Let's refrain from using this pattern.  We want to remove the `useAppWidth()` hook in the near future